order/client.go

```diff
@@ -285,6 +286,51 @@ func (i *Iter) Order() *stripe.Order {
 	return i.Current().(*stripe.Order)
 }
 
+// Return returns all or part of an order.
+// For more details see https://stripe.com/docs/api#return_order.
+func Return(id string, params *stripe.OrderReturnParams) (*stripe.OrderReturn, error) {
+	return getC().Return(id, params)
+}
+
+// Return returns all or part of an order.
+// For more details see https://stripe.com/docs/api#return_order.
+func (c Client) Return(id string, params *stripe.OrderReturnParams) (*stripe.OrderReturn, error) {
+	var body *url.Values
+	var commonParams *stripe.Params
+
+	if params != nil {
+		body = &url.Values{}
+
+		if len(params.Items) > 0 {
+			for _, item := range params.Items {
```

Comment:
> Wow, that's quite bad. Happy to pair on getting this working again, seems like we would need body to be some url.Values-like abstraction that respects ordering

+1. I'm hoping to take a stab at implementation in a couple days. I suspect that it'll largely to be a rote find and replace after designing a new abstraction, but will definitely take your pairing offer in mind ;) Thanks!